CHANGELOG.md

```diff
@@ -0,0 +1,343 @@
+# Changelog
+
+## [Unreleased](https://github.com/parallaxsecond/parsec/tree/HEAD)
+
+[Full Changelog](https://github.com/parallaxsecond/parsec/compare/0.6.0...HEAD)
+
+**Implemented enhancements:**
+
+- Investigate and define the work required for SPIFFE-based client identity management [\#232](https://github.com/parallaxsecond/parsec/issues/232)
+- Remove PKCS11 single thread lock \(\#264\) [\#277](https://github.com/parallaxsecond/parsec/pull/277) ([ionut-arm](https://github.com/ionut-arm))
+
+**Fixed bugs:**
+
+- PKCS11 multi-threading [\#264](https://github.com/parallaxsecond/parsec/issues/264)
+
+**Closed issues:**
+
+- Investigate and define the work required for compatibility with Arm Firmware Framework for Armv8-A \(FF-A\) [\#247](https://github.com/parallaxsecond/parsec/issues/247)
```

Comment:
Can we keep those "Unreleased" changes for the next release?

Reply:
I can update it for that! it just needs a flag attached to the command

Reply:
oh nice! Yes then I would be in favour of doing that for this PR and the other ones as well.